docs/src/domains/cdn/fastly.md

```diff
@@ -1,58 +1,62 @@
 ---
-title: "Fastly"
+title: "Configure your Fastly CDN"
+sidebarTitle: "Fastly"
 weight: 1
 description: |
-  In some cases you may want to opt to use a CDN such as Fastly rather than the Platform.sh router's cache. Using a CDN can offer a better time-to-first-byte for cached content across a wider geographic region at the cost of the CDN service.
 aliases:
   - "/golive/steps/fastly.html"
 ---
 
-{{% description %}}
+You can [use a CDN](./_index.md) to deliver your site's content to users more quickly.
 
-A Fastly CDN is included for Dedicated projects.
-Platform.sh doesn't offer an integrated CDN on self-service Grid projects at this time,
-but it is a common choice for customers to self-configure.
+By default, a Fastly CDN is included in Dedicated projects.
+Self-service Grid plans don't automatically include a Fastly CDN, 
+but you can set up one at any time.
 
-Launching a Platform.sh site with Fastly in front of it is nearly the same as launching normally.
-There are only a few notable differences.
-Individual apps may have their own Fastly setup instructions or additional modules.
-Consult the documentation for your app for specific details.
+## Before you begin
+
+You need:
+
+- An up-and-running Platform.sh project
+- Self-service Grid plans: a [Fastly](https://www.fastly.com/) CDN subscription
 
 {{% disable-cache CDN="Fastly" %}}
 
-## Set the Platform.sh domain on Fastly
+## 2. Dedicated: Get a DNS TXT record
+
+To use a Fastly CDN included in a Dedicated plan, 
+before going live you need to [request a DNS TXT record from support](../../overview/get-support.md#create-a-support-ticket).
+To add this DNS TXT record to your domain settings,
+contact your domain registrar.
+
+## 3. Set up your Fastly CDN
+
+To properly configure your Fastly CDN, 
+see the Fastly official documentation on [how to get started](https://docs.fastly.com/en/guides/getting-started#_basics).
```

Comment:
:question: Also a question here about whether to include the part about getting the target.